fairlearn/post_processing/threshold_optimizer.py

```diff
@@ -376,18 +376,18 @@ def _vectorized_prediction(function_dict, group_data, scores):
 
     :param function_dict: the functions that apply to various grouping attribute values
     :type function_dict: dictionary of functions
-    :param group_data: grouping attributes for each sample
-    :type group_data: list, numpy.ndarray, pandas.DataFrame, or pandas.Series
+    :param sensitive_features: grouping attributes for each sample
+    :type sensitive_features: list, numpy.ndarray, pandas.DataFrame, or pandas.Series
     :param scores: vector of predicted values
     :type scores: list, numpy.ndarray, pandas.DataFrame, or pandas.Series
     """
     # handle type conversion to ndarray for other types
-    group_data_vector = _convert_to_ndarray(group_data, MULTIPLE_DATA_COLUMNS_ERROR_MESSAGE
-                                            .format("group_data"))
+    sensitive_features_vector = _convert_to_ndarray(
+        sensitive_features, MULTIPLE_DATA_COLUMNS_ERROR_MESSAGE.format("sensitive_features"))
```

Comment:
`_KW_GROUP_MEMBERSHIP` ?

Reply:
it's not a keyword argument here, so i think it would be a bit confusing. @romanlutz ?

Reply:
If it's not a keyword argument, then it should be left as a string.